articles/quickstart/spa/_includes/_auth_service_method_description_auth0js.md

```diff
@@ -1,7 +1,30 @@
-The first noteworthy thing happening in this service is that an instance of `auth0.WebAuth` is created. The options object passed to it includes configuration for your client and domain, a response type to indicate you would like to receive an `access_token` and `id_token` after authentication, and an `audience` and `scope` which specify that authentication should be [OIDC conformant](https://auth0.com/docs/api-auth/tutorials/adoption). Also specified is the location that users should be returned to after authentication is complete. In this case, that's a route of `/callback`, which will be implemented later.
+When you set up the `AuthService` service, you create an instance of the `auth0.WebAuth` object. In that instance, you can define the following:
+* Configuration for your client and domain
+* Response type, to show that you need a user's access token and an ID token after authentication
+* Audience and scope, which specify that authentication must be [OIDC-conformant](https://auth0.com/docs/api-auth/tutorials/adoption)
+* The URL where your users are redirected to after authentication.
+::: note
+In this tutorial, the route is `/callback`, which is implemented in the Add a Callback Component step. 
```

Comment:
Yes, i left it here with the intention to add a link. I was just wondering if you use bookmarks in the content. Will correct.